dali/base/dautils.cpp

```diff
@@ -417,7 +417,20 @@ void CDfsLogicalFileName::normalizeName(const char *name, StringAttr &res, bool
         switch (c)
         {
             case '@': ct = s; break;
-            case ':': ct = nullptr; break;
+            case ':':
+            {
+                if (*(s+1) == ':')
+                {
+                    ++s;
+                    char nextc=*(s+1);
+                    if (!validFNameChar(nextc) && (!allowWild || ('*' != nextc) && ('?' != nextc) && ('{' != nextc)))
```

Comment:
Can a valid name end in '::' ?  If so then I think we would incorrectly thorw here ?

Reply:
no, that would already have been rejected as invalid.